### ISSUE TITLE
Nameless marks (or support for rhythm markers)

### ISSUE THREAD

**Issue by xsawyerx** [jliljebl/flowblade#1023]
When editing a video with music, marking the beat (or rhythm) is very helpful so I can make cuts at those points. What I wish I could do is just play the audio and press "m" to mark a beat every time I reach a good point in the audio. That way I could use those markers for cuts.

However, Flowblade requires me to give every single mark a name. Thus, I resolved to press "m" and then pressing enter. It's quite frustrating, but it's even worse because, due to #1022, I have to press *really* fast because the mark itself only triggers when I name a mark and press enter.

I wish I could have a way to mark in the audio where I want cuts without having to give it a name so it's immediately effective and I could make numerous marks to ... you know, mark the beat and rhythm.

Thanks! :)

1) Flowblade version (Help->About): 2.8.0. (flatpak)
2) MLT version (Help->Runtime Environment): 6.24.0
3) Your distribution (Ubuntu, Debian, Mint etc.): Ubuntu 21.04

**Comment by jliljebl** [comment]
Hi, I agree that the use case you described is not well served by the application. Fortunately I found a really easy fix for it and pushed it already.

I did the following:

1) If 'M' is pressed *during playback* marker is added immediately on the frame being displayed with default name 'Playback Marker'.

2) If 'M' is pressed *when is playback stopped* marker is added via dialog like previously.

3) I added a possibility to rename markers.

I believe this should enable your use case.
